speedtest.py

```diff
@@ -1171,7 +1171,7 @@ def get_config(self):
             client = get_attributes_by_tag_name(root, 'client')
 
         ignore_servers = list(
-            map(int, server_config['ignoreids'].split(','))
+            map(int, (server_config['ignoreids'].split(',') if len(server_config['ignoreids']) else []) )
```

Comment:
`map(int, server_config['ignoreids'].split(',') if server_config['ignoreids'] else [])` I suppose looks better.

Reply:
Looks nice but you don't cover the case where server_config['ignoreids']='123,421,,567'
;) (note the two commas there)

Reply:
Hmm, I agree, your solution will cover that 👍

Reply:
Btw, my solution also doesn't cover a string value like '123,notanumber, 456'. But someone has to decide where to draw the "line".

Reply:
I believe that in this case there is no need to cover all possible values. Your solution will enough.